numpy/polynomial/_polybase.py

```diff
@@ -123,6 +123,10 @@ def _line(self):
         pass
 
     @abstractmethod
+    def _interp(self):
+        pass
```

Comment:
> What do you recommend as an alternative to an API change?

> you should just remove the @abstractmethod, and implement it to raise NotImplemented

---

> Why does changing the API affect people subclassing PolynomialBase

Say I have this code:
```python
class B(ABase):
    def foo(self): return 1

b = B()
```
And the library looks like this
```python
class ABase(metaclass=ABCMeta):
    @abstractmethod
    def foo(self): pass
```
If you change it to
```   
class ABase(metaclass=ABCMeta):
    @abstractmethod
    def foo(self): pass
    @abstractmethod
    def foo(self): pass
```
Then my code using `B` will now throw an error